maquetado/styles.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding-top: 50px;
+  border-top: 8px solid #f5c85f;
+}
+
+.separator {
+  width: 3px;
+  height: 191px;
```

Comment:
no need to be so precise with this numbers, 190 would be ok. try to use even values and if possible multiples of 10. (I would have used 200px for example)